### PR TITLE
[RSDK-3364] Add Test to Validate Number of Resources from fake.json

### PR DESCRIPTION
**Pull request by 8ashar**
I added a unit test to validate the number of resources created on a server based on /etc/configs/fake.json and added two helper functions to the robottestutils package.

### CODE REVIEW

**Comment by cheukt** [comment]
prefer pr name format of "RSDK-3364 - XXXX", also can you rename the pr to be more clear about what you're adding to the codebase?

File: web/server/entrypoint_test.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	"go.viam.com/rdk/utils"
+)
+
+// numResources is the # of resources in /etc/configs/fake.json + the 2
+// expected builtin resources.
+const numResources = 19
+
+func TestNumResources(t *testing.T) {
+	logger := golog.NewTestLogger(t)
+	cfgFilename := "./../../etc/configs/fake.json"
```
Comment (cheukt) [inline web/server/entrypoint_test.go:26]: use `rdkutils.ResolveFile` to resolve this file

**Review by cheukt — COMMENTED** [review]
just commenting since I'll be OOO so don't block merge on me, but two things to address

File: web/server/entrypoint_test.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	"go.viam.com/rdk/utils"
+)
+
+// numResources is the # of resources in /etc/configs/fake.json + the 2
+// expected builtin resources.
+const numResources = 19
+
+func TestNumResources(t *testing.T) {
+	logger := golog.NewTestLogger(t)
+	cfgFilename := utils.ResolveFile("./") + "/etc/configs/fake.json"
```
Comment (njooma) [inline web/server/entrypoint_test.go:26]: you can use `ResolveFile` to resolve the entire path, not just the `./` part:

```go
cfgFilename := utils.ResolveFile("etc/configs/fake.json")
```

**Review by njooma — CHANGES_REQUESTED** [review]

**Review by njooma — APPROVED** [review]

**Pull request merged**